File: Problem5_Python/Population.py
class Population:
    def __init__(self):
        self.population = ""
        self.growth_rate = ""
        self.diff = ""

    def __init__(self, p, g):
        self.population = p
        self.growth_rate = g

    def get_population(self):
        return self.population

    def get_growth_rate(self):
        return self.growth_rate

    def get_diff(self):
        return self.diff

    def set_population(self, p):
        self.population = p

    def set_growth_rate(self, g):
        self.growth_rate = g

    def set_diff(self, s):
        self.diff = s

    def advance(self):
        self.set_diff(self.get_population() * self.get_growth_rate() - self.get_population())
        self.set_population(self.get_population() * self.get_growth_rate())

File: Problem5_Java/src/Population.java
public class Population {
    private double population;
    private double growthRate;
    private double diff;

    Population(){}

    Population(double p, double g){
        this.population = p;
        this.growthRate = g;
    }

    public double getPopulation() {
        return population;
    }

    public double getGrowthRate() {
        return growthRate;
    }

    public double getDiff() {
        return diff;
    }

    public void setGrowthRate(double growthRate) {
        this.growthRate = growthRate;
    }

    public void setPopulation(double population) {
        this.population = population;
    }

    public void setDiff(double d){
        this.diff = d;
    }

    public void advance(){
        this.diff = this.population * this.growthRate - this.population;
        this.population = this.population * this.growthRate;
    }
}
